tfx/examples/cifar10/meta_data_writer.py

```diff
@@ -0,0 +1,208 @@
+            version="v1",
+            image_width=224,
+            image_height=224,
+            image_min=0,
+            image_max=255,
+            mean=[127.5],
+            std=[127.5],
+            num_classes=10),
+}
+
+
+class MetadataPopulatorForImageClassifier(object):
+  """Populates the metadata for an image classifier."""
+
+  def __init__(self, model_file, model_info, label_file_path):
+    self.model_file = model_file
+    self.model_info = model_info
+    self.label_file_path = label_file_path
+    self.metadata_buf = None
+
+  def populate(self):
+    """Creates metadata and then populates it for an image classifier."""
+    self._create_metadata()
+    self._populate_metadata()
+
+  def _create_metadata(self):
+    """Creates the metadata for an image classifier."""
+
+    # Creates model info.
+    model_meta = _metadata_fb.ModelMetadataT()
+    model_meta.name = self.model_info.name
+    model_meta.description = ("Identify the most prominent object in the "
+                              "image from a set of %d categories." %
+                              self.model_info.num_classes)
+    model_meta.version = self.model_info.version
+    model_meta.author = "TensorFlow"
+    model_meta.license = ("Apache License. Version 2.0 "
+                          "http://www.apache.org/licenses/LICENSE-2.0.")
+
+    # Creates input info.
+    input_meta = _metadata_fb.TensorMetadataT()
+    input_meta.name = "image"
+    input_meta.description = (
+        "Input image to be classified. The expected image is {0} x {1}, with "
+        "three channels (red, blue, and green) per pixel. Each value in the "
+        "tensor is a float between {2} and {3}.".format(
+            self.model_info.image_width, self.model_info.image_height,
+            self.model_info.image_min, self.model_info.image_max))
+    input_meta.content = _metadata_fb.ContentT()
+    input_meta.content.contentProperties = _metadata_fb.ImagePropertiesT()
+    input_meta.content.contentProperties.colorSpace = (
+        _metadata_fb.ColorSpaceType.RGB)
+    input_meta.content.contentPropertiesType = (
+        _metadata_fb.ContentProperties.ImageProperties)
+    input_normalization = _metadata_fb.ProcessUnitT()
+    input_normalization.optionsType = (
+        _metadata_fb.ProcessUnitOptions.NormalizationOptions)
+    input_normalization.options = _metadata_fb.NormalizationOptionsT()
+    input_normalization.options.mean = self.model_info.mean
+    input_normalization.options.std = self.model_info.std
+    input_meta.processUnits = [input_normalization]
+    input_stats = _metadata_fb.StatsT()
+    input_stats.max = [self.model_info.image_max]
+    input_stats.min = [self.model_info.image_min]
+    input_meta.stats = input_stats
+
+    # Creates output info.
+    output_meta = _metadata_fb.TensorMetadataT()
+    output_meta.name = "probability"
+    output_meta.description = "Probabilities of the %d labels respectively." % \
```

Comment:
instead of "\", wrap the multiline statement in parenthesis.

Reply:
could you elaborate this a little bit? I'm not very sure what wrap in parenthesis mean..

Reply:
sure, we commonly avoid using "\" for newlines and instead use parenthesis to split lines.

Reply:
i see, will change it